mindmeld/components/dialogue.py

```diff
@@ -1011,6 +1027,38 @@ async def call_async(self, request, responder):
         """
         self(request, responder)
 
+    def invoke(self, request, responder):
+        """
+        Invoke slot-filling as a direct call without requiring a decorator.
+        """
+        self._invoked = True
+        kwargs = {'domain': request.domain, 'intent': request.intent}
+
+        # Set dialogue rule to auto_fill and
+        # store current rule for resetting once invoke is complete
+
+        name = self._entrance_handler.__name__
```

Comment:
probably should add a prefix or suffix with "autofill"

Reply:
This actually is the name of the follow-up handler that it redirects to once the flow is finished.

Reply:
This probably means that the handler is unique and should not be reused right?